pylint/utils/utils.py

```diff
@@ -253,6 +254,31 @@ def _check_csv(value: list[str] | tuple[str] | str) -> Sequence[str]:
     return _splitstrip(value)
 
 
+def _check_regexp_csv(value: list[str] | tuple[str] | str) -> Iterable[str]:
+    r"""Split a comma-separated list of regexps, taking care to avoid splitting
+    a regex employing a comma as quantifier, as in `\d{1,2}`."""
+    if isinstance(value, (list, tuple)):
+        yield from value
```

Comment:
Codecov message looks important since that line yields a value and apparently isn’t covered?

Reply:
No current code path sends a list of strings to this util, or the one above it (also missing coverage for the analogous line). Should we just delete these lines? Not certain--I think it makes sense to let this util be this permissive. Also helps guard against unexpected input. I'm inclined to leave as is. WDYT?

Reply:
I think taking the time to try to understand is not worth it vs just not touching it.